crates/rpc/rpc-types/src/eth/transaction/typed.rs

```diff
@@ -103,7 +119,22 @@ pub struct EIP1559TransactionRequest {
     pub input: Bytes,
     pub access_list: AccessList,
 }
-
+/// Represents an EIP-4844 transaction request
+#[derive(Debug, Clone, PartialEq, Eq, RlpEncodable, RlpDecodable)]
+pub struct Eip4844TransactionRequest {
+    pub chain_id: u64,
+    pub nonce: U64,
+    pub max_priority_fee_per_gas: U128,
+    pub max_fee_per_gas: U128,
+    pub gas_limit: U256,
+    pub kind: TransactionKind,
+    pub value: U256,
+    pub input: Bytes,
+    pub access_list: AccessList,
+    pub max_fee_per_blob_gas: u128,
+    pub blob_versioned_hashes: Vec<FixedBytes<32>>,
```

Comment:
we also need a sidecar type that contains the proofs,commitments etc

Reply:
This is a problem because BlobTransactionSidecar struct does not implement Encodable and Decodable trait, are you okay if i add them ? This will result another modification for other struct too

Reply:
we need to duplicate that struct inside rpc-types and can't reuse the existing one

Reply:
> we need to duplicate that struct inside rpc-types and can't reuse the existing one

Amazing! I've just done that. Thank you so much !